nibabel/nicom/tests/test_utils.py

```diff
@@ -67,3 +68,34 @@ def test_find_private_section_real():
     assert_equal(find_private_section(ds, 0x11, 'near section'), 0x1300)
     ds.add_new((0x11, 0x15), 'LO', b'far section')
     assert_equal(find_private_section(ds, 0x11, 'far section'), 0x1500)
+
+
+def test_tm_to_seconds():
+    for str_val in ('', '1', '111', '11111', '111111.', '1111111', '1:11',
+                    ' 111'):
+        assert_raises(ValueError, tm_to_seconds, str_val)
+    assert_almost_equal(tm_to_seconds('01'), 60*60)
+    assert_almost_equal(tm_to_seconds('0101'), 61*60)
+    assert_almost_equal(tm_to_seconds('010101'), 61*60 + 1)
+    assert_almost_equal(tm_to_seconds('010101.001'), 61*60 + 1.001)
+    assert_almost_equal(tm_to_seconds('01:01:01.001'), 61*60 + 1.001)
+
+
+def test_tm_rt():
+    for tm_val in ('010101.00000', '010101.00100', '122432.12345'):
+        assert_equal(tm_val, seconds_to_tm(tm_to_seconds(tm_val)))
+
+
+def test_as_to_years():
+    assert_equal(as_to_years('1'), 1.0)
```

Comment:
Is just `1` or `2` valid?  Does it reliably refer to a year?

Do strings have to be zero padded to the left (as in `001Y`)?  Standard seems to suggest that - but maybe not so in practice?

Maybe worth testing strings like `061Y` and `002W`.